src/main/java/me/svdev/mscategory/service/CategoryService.java

```diff
@@ -0,0 +1,72 @@
+package me.svdev.mscategory.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.slf4j.Slf4j;
+import me.svdev.mscategory.dao.entity.CategoryEntity;
+import me.svdev.mscategory.dao.repository.CategoryRepository;
+import me.svdev.mscategory.exception.NotFoundException;
+import me.svdev.mscategory.mapper.CategoryMapper;
+import me.svdev.mscategory.mapper.SubcategoryMapper;
+import me.svdev.mscategory.model.request.CategoryRequest;
+import me.svdev.mscategory.model.response.CategoriesResponse;
+import me.svdev.mscategory.model.response.CategoryResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import static lombok.AccessLevel.PRIVATE;
+import static me.svdev.mscategory.exception.ExceptionMessages.NOT_FOUND_ERROR_WITH_ID;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+@FieldDefaults(level = PRIVATE, makeFinal = true)
+public class CategoryService {
+
+    CategoryRepository categoryRepository;
+
+    public CategoriesResponse getCategories() {
+        log.info("ActionLog.getCategories.start");
+        var categories = new CategoriesResponse(categoryRepository.findAll().stream().map(CategoryMapper::mapEntityToResponse).toList());
+        log.info("ActionLog.getCategories.end");
+        return categories;
+    }
+
+    @Transactional(rollbackFor = {RuntimeException.class})
```

Comment:
1. Remove @Transactional
2. rollbackFor = RuntimeException.class this is invalid argument